Source/WebCore/html/track/TextTrackCue.idl

```diff
@@ -33,17 +35,17 @@
     LegacyFactoryFunctionEnabledBySetting=GenericCueAPIEnabled,
     Exposed=Window
 ] interface TextTrackCue : EventTarget {
-    [CallWith=CurrentDocument] constructor(double startTime, double endTime, DocumentFragment cueNode);
-
-    readonly attribute TextTrack track;
+    readonly attribute TextTrack? track;
 
     attribute [AtomString] DOMString id;
     attribute double startTime;
+    // FIXME: Add 'unrestricted' to 'endTime' to align with Web-Spec.
     attribute double endTime;
     attribute boolean pauseOnExit;
 
     attribute EventHandler onenter;
     attribute EventHandler onexit;
 
+    [CallWith=CurrentDocument] constructor(double startTime, double endTime, DocumentFragment cueNode);
```

Comment:
This should probably have a FIXME to make `endTime` unrestricted

Reply:
Will look again in web-spec, might be oversight on my end.

Reply:
Done. :-)